docs/modules/ROOT/pages/discovery.adoc

```diff
@@ -0,0 +1,57 @@
+:clusterName: simple-derby-druid
+:namespace: stackable
+:routerPort: 8888
+
+= Discovery Profiles
+
+The Stackable Operator for Apache Druid creates a single discovery profile that allows access to multiple Druid services which are published into the Kubernetes cluster as a
+https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#configmap-v1-core[`ConfigMap`] object.
+
+== Profiles
+
+Given the following Druid cluster:
+
+[subs="attributes"]
+----
+apiVersion: druid.stackable.tech/v1alpha1
+kind: DruidCluster
+metadata:
+  name: {clusterName} # <1>
+  namespace: {namespace} # <2>
+spec:
+  [...]
+----
+<1> Name of the Druid cluster which is the name of the `ConfigMap` created for this discovery profile.
+<2> Namespace of the `ConfigMap` created for this discovery profile.
+
+The published `ConfigMap` name and namespace are equal to the cluster name and namespace. In this case `{namespace}/{clusterName}`.
+
+=== Default
+
+This profile allows access to the Druid cluster from inside the Kubernetes cluster, and connects directly to the router NodePort `Service`.
```

Comment:
Talking about a connection here is confusing, we haven't even explained what the discovery config map is and that it is about connecting to stuff really

Reply:
Also "Default" is a bit weird if there are no other profiles.

I understand that we want to keep it similar to other docs, but if I read this in isolation it is more confusing than helpful